src/server/services/status.js

```diff
@@ -11,13 +11,25 @@ export async function getAllStatus() {
 }
 
 /**
+ * Fetch a service by its name.
+ *
+ * @param  {String} id
+ * @return {Promise}
+ */
+export async function getStatusGroupedByName() {
+  return await new Status().query(qb => qb.groupBy('name')
```

Comment:
Looks like a basic query, doesn't really matter if we use the query builder. I am fine with any approach.